src/utils.ts

```diff
@@ -89,7 +89,7 @@ addErrors(ajv);
 ajv.addFormat('address', {
   validate: (value: string) => {
     try {
-      return isAddress(value);
+      return /^0x[0-9a-fA-F]{62,64}$/.test(value) || isAddress(value);
```

Comment:
i think it is better to do `isAddress` first, as it is our primary check

```suggestion
      return isAddress(value) || /^0x[0-9a-fA-F]{62,64}$/.test(value);
```

Reply:
This may allow stark addresses here https://github.com/snapshot-labs/snapshot.js/blob/master/src/schemas/alias.json#L11 should be fine?

Reply:
Should be fine I think.

Reply:
> i think it is better to do `isAddress` first, as it is our primary check

Will do more tests to make sure that it does not throw an error, and move on to the `catch` block, before checking for starknet address